src/image_copy/main.go

```diff
@@ -69,44 +73,59 @@ func getTags(dstTag string, updateMajor bool) []string {
 	return tags
 }
 
+func execCmd(name string, args ...string) string {
+	cmd := exec.Command(name, args...)
+	fmt.Println("\n> ", name, strings.Join(args, " "))
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Println(out)
+		log.Fatal(err.Error())
+	}
+	return string(out)
+}
+
 func mustHaveArchitecture(imageTag, platform, expectedArch string) {
-	fmt.Printf("verifying binaries in '%s' from '%s' for platform '%s' are '%s'\n", binFolder, imageTag, platform, expectedArch)
+	fmt.Printf("\nverifying binaries in '%s' from '%s' for platform '%s' are '%s'\n", binFolder, imageTag, platform, expectedArch)
 
-	cmd := exec.Command(
+	// create local container from image
+	containerId := base64.RawURLEncoding.EncodeToString([]byte(fmt.Sprintf("%v-%v", platform, time.Now().UnixMilli())))
```

Comment:
could just be a random string?

Reply:
Right; good catch!

Reply:
This was my way of generating a random string without a lot of ceremony. Is there a shorter/better way?

Reply:
`uuid.NewString()`?

Reply:
Right; I would have to add that as a dependency. Do you feel strongly about this approach?

Reply:
Okay, changed.

Reply:
no big deal, fine either way